src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs

```diff
@@ -68,16 +68,17 @@ public override Task<bool> CanLaunchAsync(DebugLaunchOptions launchOptions)
         /// <summary>
         /// Helper returns the correct debugger engine based on the targeted framework
         /// </summary>
-        public static Guid GetManagedDebugEngineForFramework(string targetFramework)
+        public static Guid GetManagedDebugEngineForFramework(string targetFramework, bool mixedMode)
         {
             // The engine depends on the framework
             if (IsDotNetCoreFramework(targetFramework))
             {
+                if (mixedMode) throw new Exception(VSResources.UnmanagedDebuggingNetCoreNotSupported);
```

Comment:
Thanks, I'll update this PR to do that for net core. Is that also the right way for net framework, or is returning the MixedNativeAndManagedEngine value the right way?